Intel NUC Getting Started secure boot note (#1817)
balena-io/docs: shared/getting-started/getDeviceOnDash/intel-nuc.md

Now connect up the power supply and turn the device on. The power button may be a small round button on the top of the device or a square button on the front.

> Press the F10 key while the **BIOS** is loading in order to enter the boot menu. Next, select the `UEFI : USB` (or `resinOS`) option from the boot menu so that the device will boot from your USB drive.

Reviewer: @kb2ma is it still being called resinOS on boot menu?

Reviewer: If yes, then just as a side task. Ask the balenaOS team if they are working towards resolving that and on track to changing it to balenaOS

Author: Seeing is believing. I will followup with an issue in balena-os/balena-intel repo.

![boot-menu](https://user-images.githubusercontent.com/2944058/114696143-59ee4580-9cea-11eb-9b30-87dbf4bb8735.jpg)

Author: Expecting a response on the 'resinOS' name question in balena-os/balena-intel#362.

Reviewer: Thanks for creating the issue @kb2ma

Once the device boots, you should see it pop up on your {{ $names.company.lower }} dashboard. It will immediately go into a `flashing internal media` state. This means that the device is flashing the {{ $names.os.lower }} onto your internal flash media.

> __Note:__ You may see an error message when the device boots with the text, "Image Authorization Fail". This message appears when Secure Boot is enabled. Follow the steps in the linked Intel [support document](https://www.intel.com/content/www/us/en/support/articles/000038401/intel-nuc/intel-nuc-kits.html) to access the BIOS setup screen and disable secure boot. After saving that change, once again press the F10 key when the NUC reboots to enter the boot menu, and select to boot from USB/resinOS.

Reviewer: ```suggestion
__Note:__ You might encounter an error message when the device boots with the text, "Image Authorization Fail". This message appears when Secure Boot is enabled. Follow the steps present in the [Intel support document](https://www.intel.com/content/www/us/en/support/articles/000038401/intel-nuc/intel-nuc-kits.html) to access the BIOS setup screen and disable secure boot. After saving, press the F10 key once again when the NUC reboots to enter the boot menu & select to boot from USB/resinOS.
```

Author: Replacement text looks good, except my English teacher would rise from the grave to whack me on the knuckles for the ampersand. If you don't mind I'll just remove the comma after 'menu', like this:

```
After saving, press the F10 key once again when the NUC reboots to enter the boot menu and select to boot from USB/resinOS.
```

Reviewer: Sure it's all good.

Author: @vipulgupta2048, I squashed in the update for the Note as discussed. All comments should be addressed at this point.
